matRad_calcGeoDists.m

```diff
@@ -95,14 +98,22 @@
 ix = radDepthIx(subsetMask);
 
 % return radial distances squared
-if nargout > 1
-    rad_distancesSq = rad_distancesSq(subsetMask);
-end
+rad_distancesSq = rad_distancesSq(subsetMask);
 
 % return x & z distance
-if nargout > 2
-   isoLatDistsX = latDistsX(subsetMask)./rot_coords_temp(subsetMask,2)*SAD;
-   isoLatDistsZ = latDistsZ(subsetMask)./rot_coords_temp(subsetMask,2)*SAD; 
-end
-
+% if nargout > 2
+%    isoLatDistsX = latDistsX(subsetMask)./rot_coords_temp(subsetMask,2)*SAD;
+%    isoLatDistsZ = latDistsZ(subsetMask)./rot_coords_temp(subsetMask,2)*SAD; 
+% end
 
+if nargout > 4
```

Comment:
I still don't fully understand the logic here. If nargout > 2, you need latDistX, right? So, why not always do the latDistX = latDistX(subsetMask) as soon as nargout > 2?
You don't even need the nargout > 4 then. But I also don't understand why in the previous version they were returned withoutthe subindexing. Or am I just unreasonably confused?

Reply:
I think the initial purpose of this if statement was to compute isoLatDistsX and isoLatDistsZ only when the user asks for it. But I have to admit that I am also confused now. I will get rid of the if statement in the function then.